graylog2-rest-client/src/main/java/org/graylog2/restclient/models/alerts/AlertCondition.java

```diff
@@ -37,7 +37,8 @@
 
     public enum Type {
         MESSAGE_COUNT,
-        FIELD_VALUE
+        FIELD_VALUE,
+        FIELD_STRING_VALUE
```

Comment:
Should be `FIELD_CONTENT_VALUE` to be consistent with the actual name of the condition.